core/session.ts

```diff
@@ -277,6 +294,19 @@ export class Session {
     return assertion;
   }
 
+  public declare(dataset: FullyQualifiedName): Declaration {
```

Comment:
Why not `Resolvable`?

Reply:
For declarations, I don't think we should ever let users just pass a table's name, because if they do, we don't really know what schema to use. We could use the schema(s) defined in `dataform.json`, but then the question is: what's the correct behaviour when the user asks for schema suffixes? I think for clarity / user understandability sake, forcing them to always specify the schema is the right thing to do.